### PR TITLE
Decision tracking with GBF Search

### PR DESCRIPTION
**Pull request by rqk2k**
### General description
Fixing the permission on editing decisions as well as adding new decision tracking editing tagging (GBF targets and Goals) and search using the solr index.

### Designs
N/A

### Testing instructions
- The solr search has been tested and validated on the dev environment using the URL: https://www.cbddev.xyz/decisions/search/
- The tagging functionality has been tested and validaded on the URL: https://www.cbddev.xyz/decisions/cop/15

## Checklist before merging
* [X]  Branch name / PR includes the related Jira ticket Id.
* [ ]  Tests to check core implementation / bug fix added. (Has been tested on DEV environment).
* [X]  All checks in Continuous Integration workflow pass.
* [X]  Feature functionally tested by reviewer(s).
* [ ]  Code reviewed by reviewer(s). - In progress
* [ ]  Documentation updated (README, CHANGELOG...) (if required) (N/A)

### CODE REVIEW

File: app/data/gbf-targets/goals.json [hunk 1]
```diff
@@ -0,0 +1,82 @@
+[
```
Comment (randyhoulahan) [inline app/data/gbf-targets/goals.json:1]: why not from the api?

File: app/images/gbf-targets/gbf-23-64.png
Comment (randyhoulahan) [inline app/images/gbf-targets/gbf-23-64.png:1]: are these official?  please share source in ref if i missed it.  I am using a blue unofficial version.

**Review by randyhoulahan — APPROVED** [review]

**Review by randyhoulahan — APPROVED** [review]

File: app/data/gbf-targets/goals.json (hunk 1 above)
Comment (rqk2k) [reply app/data/gbf-targets/goals.json:1]: This was a old file, the actual page is getting from the API.

Comment (sbilodeau) [reply app/data/gbf-targets/goals.json:1]: can you delete?

File: app/routes/decisions.js [hunk 2]
```diff
@@ -21,7 +22,7 @@ app.config(['$routeProvider', '$locationProvider', function($routeProvider, $loc
     
     $routeProvider
     .when('/',                                    { ...mapView(redirectView),           resolve: { } })
-    .when('/search',                              { ...mapView(searchView) ,            resolve: { user : currentUser() }, reloadOnSearch : false } )
+    .when('/search',                              { ...mapView(vueViewWrapper),         resolve: { ...decisionSearch, user : currentUser() }, reloadOnSearch : false } )
```
Comment (sbilodeau) [inline app/routes/decisions.js:25]: can you delete the old search at lien #8 and delete the old search files

**Review by sbilodeau — CHANGES_REQUESTED** [review]
2-3 comments. please update

File: app/routes/decisions.js (hunk 2 above)
Comment (rqk2k) [reply app/routes/decisions.js:24]: done.

File: app/data/gbf-targets/goals.json (hunk 1 above)
Comment (randyhoulahan) [reply app/data/gbf-targets/goals.json:1]: @sbilodeau I assisted here.  Tried to call with axios, created and mounted.  Getting strange error I cannot resolve (quick).  I think it angualar-vue package.  I will put this through then figure out the problem why I can't get axios to work.

**Pull request merged**